llvm/test/CodeGen/AMDGPU/vni8-across-blocks.ll

```diff
@@ -1982,5 +350,56 @@ bb.2:
   ret void
 }
 
-declare i32 @llvm.amdgcn.workitem.id.x()
+define amdgpu_kernel void @repeat_successor(i32 %in, ptr addrspace(1) %src1, ptr addrspace(1) %src2, ptr addrspace(1) nocapture %dst) {
```

Comment:
@arsenm Is this what you meant by "predecessor block has a repeated successor" test?

Reply:
Yes, you have to have a switch with the same dest repeated twice, which gets 2 phi entries